CoreAutomata/TransformationExtensions.cs

```diff
@@ -79,21 +74,18 @@ public static Region TransformFromImage(this Region Region)
             // Screen -> Image
             var scale2 = ScreenToImageScale();
 
-            if (scale1 == null && scale2 == null)
-            {
-                return Region;
-            }
-
-            var scale = (1 / (scale1 ?? 1)) * (1 / (scale2 ?? 1));
+            var scale = scale1 * (scale2 ?? 1);
 
-            return Region * scale;
+            return Region * (1 / scale);
         }
 
-        static double? ScriptToScreenScale()
+        const double DontScale = 1;
+
+        static double ScriptToScreenScale()
         {
             if (GameAreaManager.ScriptDimension == null)
             {
-                return null;
+                return DontScale;
```

Comment:
Why don't you also do this for ScreenToImageScale()?

Reply:
`ScreenToImageScale` is used by `ScreenshotManager` to decide if the screenshot needs to be resized. if `null`, don't resize. I don't want to check `scale == DontScale` due to the issue I wrote about in floating-point equality check.

Reply:
Oh, I think I get what you're saying now.
You meant `scale ?? DontScale` instead of `scale ?? 1` right? I'll make the change.